### ISSUE TITLE
the effect of pretrained model

### ISSUE THREAD

**Issue by lxy2017** [yangxy/GPEN#8]
Hi yangxy
Thanks for GPEN
What is the comparison between the effect of without/with stylegan2 pretrained model?  
no pretrained model  vs   stylegan2 pretrained model ft

**Comment by yangxy** [comment]
In our experiments, if the degradation is not that severe, no pre-trained model can achieve comparable results, which is somewhat not consistent with the idea claimed in our paper. However, in cases where the face is severely degraded such as 64x FSR, no pre-trained model can hardly produce any clear results, while GPEN still works well. 

Hope this helps.

**Comment by lxy2017** [comment]
> In our experiments, if the degradation is not that severe, no pre-trained model can achieve comparable results, which is somewhat not consistent with the idea claimed in our paper. However, in cases where the face is severely degraded such as 64x FSR, no pre-trained model can hardly produce any clear results, while GPEN still works well.
> 
> Hope this helps.

Thanks for answering.
noise add vs noise cat, should I retrain a stylegan2 model with noise cat?

**Comment by yangxy** [comment]
> > In our experiments, if the degradation is not that severe, no pre-trained model can achieve comparable results, which is somewhat not consistent with the idea claimed in our paper. However, in cases where the face is severely degraded such as 64x FSR, no pre-trained model can hardly produce any clear results, while GPEN still works well.
> > Hope this helps.
> 
> Thanks for answering.
> noise add vs noise cat, should I retrain a stylegan2 model with noise cat?

Yes if you want to reproduce our results. Actually, the noise-add version of GPEN can work quite well too, in such case you can simply adopt a well-trained StyleGAN.

**Comment by lxy2017** [comment]
> > > In our experiments, if the degradation is not that severe, no pre-trained model can achieve comparable results, which is somewhat not consistent with the idea claimed in our paper. However, in cases where the face is severely degraded such as 64x FSR, no pre-trained model can hardly produce any clear results, while GPEN still works well.
> > > Hope this helps.
> > 
> > 
> > Thanks for answering.
> > noise add vs noise cat, should I retrain a stylegan2 model with noise cat?
> 
> Yes if you want to reproduce our results. Actually, the noise-add version of GPEN can work quite well too, in such case you can simply adopt a well-trained StyleGAN.

Thanks
Lf is similar to the perceptual loss [19] but it is based on the discriminator rather than the pre-trained VGG network to fit our task.
If I use pre-trained VGG , the effect will be similar?

**Comment by yangxy** [comment]
> > > > In our experiments, if the degradation is not that severe, no pre-trained model can achieve comparable results, which is somewhat not consistent with the idea claimed in our paper. However, in cases where the face is severely degraded such as 64x FSR, no pre-trained model can hardly produce any clear results, while GPEN still works well.
> > > > Hope this helps.
> > > 
> > > 
> > > Thanks for answering.
> > > noise add vs noise cat, should I retrain a stylegan2 model with noise cat?
> > 
> > 
> > Yes if you want to reproduce our results. Actually, the noise-add version of GPEN can work quite well too, in such case you can simply adopt a well-trained StyleGAN.
> 
> Thanks
> Lf is similar to the perceptual loss [19] but it is based on the discriminator rather than the pre-trained VGG network to fit our task.
> If I use pre-trained VGG , the effect will be similar?

To be honest, a single L1 loss leads to quite nice results.

**Comment by yangxy** [comment]
> > > > > In our experiments, if the degradation is not that severe, no pre-trained model can achieve comparable results, which is somewhat not consistent with the idea claimed in our paper. However, in cases where the face is severely degraded such as 64x FSR, no pre-trained model can hardly produce any clear results, while GPEN still works well.
> > > > > Hope this helps.
> > > > 
> > > > 
> > > > Thanks for answering.
> > > > noise add vs noise cat, should I retrain a stylegan2 model with noise cat?
> > > 
> > > 
> > > Yes if you want to reproduce our results. Actually, the noise-add version of GPEN can work quite well too, in such case you can simply adopt a well-trained StyleGAN.
> > 
> > 
> > Thanks
> > Lf is similar to the perceptual loss [19] but it is based on the discriminator rather than the pre-trained VGG network to fit our task.
> > If I use pre-trained VGG , the effect will be similar?
> 
> To be honest, a single L1 loss leads to quite nice results.

plus LA of course

**Comment by lxy2017** [comment]
> > > > > > In our experiments, if the degradation is not that severe, no pre-trained model can achieve comparable results, which is somewhat not consistent with the idea claimed in our paper. However, in cases where the face is severely degraded such as 64x FSR, no pre-trained model can hardly produce any clear results, while GPEN still works well.
> > > > > > Hope this helps.
> > > > > 
> > > > > 
> > > > > Thanks for answering.
> > > > > noise add vs noise cat, should I retrain a stylegan2 model with noise cat?
> > > > 
> > > > 
> > > > Yes if you want to reproduce our results. Actually, the noise-add version of GPEN can work quite well too, in such case you can simply adopt a well-trained StyleGAN.
> > > 
> > > 
> > > Thanks
> > > Lf is similar to the perceptual loss [19] but it is based on the discriminator rather than the pre-trained VGG network to fit our task.
> > > If I use pre-trained VGG , the effect will be similar?
> > 
> > 
> > To be honest, a single L1 loss leads to quite nice results.
> 
> plus LA of course

use a pretrained model for discriminator?

**Comment by lxy2017** [comment]
> > > > > > In our experiments, if the degradation is not that severe, no pre-trained model can achieve comparable results, which is somewhat not consistent with the idea claimed in our paper. However, in cases where the face is severely degraded such as 64x FSR, no pre-trained model can hardly produce any clear results, while GPEN still works well.
> > > > > > Hope this helps.
> > > > > 
> > > > > 
> > > > > Thanks for answering.
> > > > > noise add vs noise cat, should I retrain a stylegan2 model with noise cat?
> > > > 
> > > > 
> > > > Yes if you want to reproduce our results. Actually, the noise-add version of GPEN can work quite well too, in such case you can simply adopt a well-trained StyleGAN.
> > > 
> > > 
> > > Thanks
> > > Lf is similar to the perceptual loss [19] but it is based on the discriminator rather than the pre-trained VGG network to fit our task.
> > > If I use pre-trained VGG , the effect will be similar?
> > 
> > 
> > To be honest, a single L1 loss leads to quite nice results.
> 
> plus LA of course

Thanks!
If you use a pre-trained model for discriminator?
